src/modules/navigator/mission_params.c

```diff
@@ -167,3 +152,16 @@ PARAM_DEFINE_FLOAT(MIS_YAW_ERR, 12.0f);
  * @group Mission
  */
 PARAM_DEFINE_FLOAT(MIS_PD_TO, 5.0f);
+
+/**
+ * Landing abort min altitude
+ *
+ * Minimum altitude above landing point that the vehicle will climb to after an aborted landing.
+ * Then vehicle will loiter in this altitude until further command is received.
+ * Only applies to fixed-wing vehicles.
+ *
+ * @unit m
+ * @min 0
+ * @group Mission
+ */
+PARAM_DEFINE_INT32(MIS_LND_ABRT_ALT, 30);
```

Comment:
@sfuhrer why an int instead of float?

Reply:
Safes a little bit of flash I thought - in which case would you need a sub-integer resolution?